### ISSUE TITLE
LSB Replacement?

### ISSUE THREAD

**Issue by MK-RD** [qtranslate/qtranslate-xt#983]
In a current project I have been able to completely replace the buttons (LSB) for switching the language with a central menu.

Maybe you like the idea and we could add somthing simular to the plugin?

![Desktop](https://user-images.githubusercontent.com/79551145/111549259-274d3d80-877c-11eb-92f0-c9c0568c7bc9.png)

![Landscape](https://user-images.githubusercontent.com/79551145/111547956-3cc16800-877a-11eb-9a0c-140e98ef9e37.png)

![Portrait](https://user-images.githubusercontent.com/79551145/111548910-b4dc5d80-877b-11eb-89b4-781ce9a84af8.png)

**Comment by herrvigg** [comment]
Unless I'm missing something, we have already that functionality in qTranslate with the Single Language Mode.
You can check it in settings / languages / advanced / editor mode -> Single Language Mode.

This is meant to be a fallback mode in case the LSB don't work. It's also the same principle we use for Gutenberg currently.

**Comment by MK-RD** [comment]
You are probably right that a combination of these two modes may not be easily understood by the user.

So for my specific case, I'll be adding an extra menu to enable direct language switching in "classic mode".

**Comment by herrvigg** [comment]
But what's the difference with the Single Language Mode? This works already with the Classic Editor. The language list comes the standard admin menu.

**Comment by MK-RD** [comment]
That's right, but you can't edit in your native language adn we currently only use the "Classic mode" in our projects!
In addition, the buttons flood the administration area, the future appearance of which will be determined by Wordpress.

If you are interested, I can give you an example today.

**Comment by MK-RD** [comment]
**la démonstration**

![en](https://user-images.githubusercontent.com/79551145/111803023-ee71ad80-88ce-11eb-91dc-d758a784876b.png)

![ar](https://user-images.githubusercontent.com/79551145/111803026-ef0a4400-88ce-11eb-8901-e882864afc56.png)

**Comment by MK-RD** [comment]
Example: Available for one week only 
https://www.mk-research.com/qtx_mu-plugin.zip

Usage: Must Use Plugins
https://wordpress.org/support/article/must-use-plugins/

**Comment by MK-RD** [comment]
@herrvigg Just to be on the safe side, I would like to ask whether this form of editing (LSB) will be supported in the future?

**Comment by herrvigg** [comment]
New functionalities are welcome but it's currently hard to judge what it brings. Can you show a video illustrating this?

As written earlier, it looks like the "single language mode" functionality. In that case it's not a replacement of the LSB which allow the edition of multiple languages at the same time before saving a post. The complexity of the two modes is very different (LSB is more complex than single language mode).

> That's right, but you can't edit in your native language

What do you mean? I don't see why this would not be possible with the current solution. Switch WP language to native language.

**Comment by herrvigg** [comment]
With this new functionality, do you have to save the language every time you switch? In that case I would not call it a replacement for LSB which allow to edit multiple languages before saving.

**Comment by MK-RD** [comment]
> 
> 
> With this new functionality, do you have to save the language every time you switch? In that case I would not call it a replacement for LSB which allow to edit multiple languages before saving.

Of course not!
